pkg_defs/Chandra.Maneuver/meta.yaml

```diff
@@ -40,4 +42,4 @@ test:
 
 
 about:
-  home: git@github.com:sot/Chandra.Maneuver
+  home: https://github.com/sot/Chandra.Maneuver
```

Comment:
This looks fine, though I think I left some of these with ssh style because the auth worked better for me.  Will see if that just needs auth fixes for the aca user.

Reply:
That's a good question... the automated build uses http because it is easier to authenticate. If you check, I jumped through some hoops to do that. I personally prefer ssh.

Reply:
Ah, that's because the `about: home` tag is being (mis-)used as the clone URL instead of the intent of being the project home.  We might consider populating the `about: dev_url` tag instead which is closer to the idea of a clone URL. 

The curious thing is that this does work for me on my mac, and I have 2FA so generally speaking https has presented problems (requiring a token).  I'm not exactly sure what is happening under the hood to allow this https: access to successfully clone a repo.  I just tested `git clone https://github.com/sot/maude.git` for aldcroft@mac (works) and aca@kady (fails). 

Anyway I'll just revert this to reduce problems.